mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -80,13 +80,8 @@ def case_open(self):
 
         for case in self:
             values = {'active': True}
-
-            stages = self.env['mgmtsystem.action.stage']
-            values['stage_id'] = stages.search([
-                ['is_ending', '=', False],
-                ['is_starting', '=', False]
-            ]).id
-
+            values['stage_id'] = self.env.ref(
+                'mgmtsystem_action.stage_open').id
```

Comment:
It's not equivalent code, since you could use custom Stages, with no XML Id.
I suggest instead to add `limit=1` to the original `search()`.